docs/packages/utils.rst

```diff
@@ -1,7 +1,7 @@
 Snorkel Utils Package
 ---------------------
 
-None
+General modeling utilities shared across Snorkel.
```

Comment:
I would be hesitant to bucket them purely under modeling— even broader `ML utilities` could be safer